build/common.sh

```diff
@@ -171,6 +171,10 @@ function kube::build::verify_prereqs() {
 function kube::build::docker_available_on_osx() {
   if [[ -z "${DOCKER_HOST}" ]]; then
     kube::log::status "No docker host is set. Checking options for setting one..."
+    if [[ -a "/Applications/Docker.app/Contents/MacOS/Docker" ]]; then
```

Comment:
I don't think so http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html tells that it means `True if FILE exists.` any other suggestions?